bootstrap.sh

```diff
@@ -9,5 +9,3 @@ carthage bootstrap --platform iOS --color auto --cache-builds
 cd lockbox-ios
 tar xvfz binaries.tar.gz
 cd ..
-
-source 'scripts/update-datastore'
```

Comment:
Weeeee 🕺

@sashei Should references also be removed from the `buddybuild_prebuild` and `gitignore` files?